php/WC_Filter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+defined( 'ABSPATH' ) or exit;
+add_action( 'plugins_loaded', 'wc_filter_orders_by_bitcoin_address' );
+
+class WC_Filter_Orders_By_Address {
+	protected static $instance;
+	public function __construct() {
+		if ( is_admin() ) {
+			// add bulk order filter for exported / non-exported orders
+			add_action( 'restrict_manage_posts', array( $this, 'filter_orders_by_address') , 20 );
+			add_filter( 'request',               array( $this, 'filter_orders_by_query' ) );	
+		}
+	}
+	public function filter_orders_by_address() {
+		global $typenow;
+
+		// add_settings_error('option_notice', 'option_notice', 'Hello', 'error');
+		if ( 'shop_order' === $typenow ) {
+			?>
+			<input type='name' placeholder='Filter bitcoin payments' name='filter_by_address'>
+			<select name="type_query" id="dropdown_type_query">
+				<option value="btc_address">
```

Comment:
Filtering by the btc_address, btc_txid, and bch_txid is working correctly. Filter by bch_address should also be added.

Reply:
added bch_address as an option.